src/components/Catalog.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import Movie from './Movie'
+class Catalog extends Component {
+    constructor() {
+        super()
+        this.state = {
+            forSarch: '',
+            Budget: 9,
+            catalog: [
+                { id: 0, isRented: false, title: "Tarzan", year: 1999, img: "https://vignette.wikia.nocookie.net/disney-fan-fiction/images/4/42/Tarzan_2004_cover.jpg/revision/latest?cb=20140331030811", descrShort: "Tarzan was born into wealth but raised into incredible misfortune. Shiprweck, parents mauled by a jaguar. Luckily, a troop of gorillas took him in, but the Big Daddy gorilla never took a liking to him. That is, until the end when it's too late. Why is it too late? Watch and find out." },
```

Comment:
you are supposed to have only one source of truth. This means, data written only once in your application. You have the catalog written twice. Instead, it should be passed down as props and not re-written